src/native/libs/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ if (CLR_CMAKE_TARGET_UNIX OR CLR_CMAKE_TARGET_BROWSER OR CLR_CMAKE_TARGET_WASI)
 
     if (CLR_CMAKE_TARGET_BROWSER OR CLR_CMAKE_TARGET_WASI)
         set(GEN_SHARED_LIB 0)
-        set(STATIC_LIB_DESTINATION .)
+        set(STATIC_LIB_DESTINATION ${CLR_ARTIFACTS_BIN_DIR}/native/net${CMAKE_NET_CORE_APP_CURRENT_VERSION}-browser-${CMAKE_BUILD_TYPE}-wasm/)
```

Comment:
It is odd to see such a hardcoded output path. It deserved a comment on why it is needed.

Reply:
I'm not confident with this change. 

The problem, I think, is caused by the fact that `src/native/libs` are built as part of coreRun. Because it's statically linked and because we are doing CMake include from coreRun.

Reason why I want the outputs in the same directory as usual, is that I want the liveBuild to work.

Advice appreciated, I struggle with CMake.

Reply:
I think I get it now, I need to build subset `libs.native` to get those into correct folder.

Reply:
I tried to set it on `add_subdirectory()` second parameter instead. I don't know if that's better.
I pushed broken version. We will look at it with @akoeplinger tomorrow.

Reply:
That's ^^ not right. I will continue iterating here.
I split easier parts into https://github.com/dotnet/runtime/pull/120359